fusesoc/capi1/section.py

```diff
@@ -490,6 +490,21 @@ def __init__(self, items=None):
         if items:
             self.load_dict(items)
 
+class TrellisSection(ToolSection):
+
+    TAG = 'trellis'
+
+    def __init__(self, items=None):
+        super(TrellisSection, self).__init__()
+
+        self._add_member('nextpnr_options', StringList, "nextpnr options")
+        self._add_member('yosys_synth_options' , StringList, "Additional options for the synth_* commands in yosys")
+        self._add_member('top_module', str, 'RTL top-level module')
+        self.top_module = 'orpsoc_top'
```

Comment:
We don't want orpsoc_top as the default top_module. This is only a remain in some of the backends for backwards compatibility

Reply:
Ah, think I added this while trying to get the old SoC project to build.